### ISSUE TITLE
[Question] Log file location

### ISSUE THREAD

**Issue by zxsix** [Laharah/deluge-FileBotTool#15]
I have installed the tool correctly on a Synology NAS.  I have right-clicked on a torrent and tested my formatting, both dry runs, and actual renaming.  That works.  I then set up rules and filebot did it's thing upon completion of the torrent.

My question is, what is the location of any log files that I can review?
i.e. what torrents finished last night?  did it process and hardlink successfully or did it abort due to not matching the file to a movie on themoviedb?

**Comment by Laharah** [comment]
Right now there isn't any Filebot specific log file. When I wrote the plugin it was assumed that only moves would be done, so the feedback would always be visible directly in deluge. FilebotTool logs are output to the deluge logging system, I believe that successful torrents are output at the `info` log level, while non-matched torrents will be logged at the `warning` or `error` log level.

**Comment by zxsix** [comment]
thanks.  i plan to let it trigger upon completion of torrent download and use hardlinks.
just want to keep ane eye on any that might get skipped.

**Comment by Laharah** [comment]
yeah, it does need some other feedback mechanism for actions that don't result in a file move. I'll mark this for enhancement and keep it in mind should I find time to work on the plugin.

**Comment by Laharah** [comment]
@zxsix 
as an aside, was it difficult to get the plugin running on synology? I've had a few people ask. Did you have to do anything special?

**Comment by zxsix** [comment]
There may have been a dependency that it wanted to install, but nothing harder than clicking a few next buttons.
